packages/stripe_checkout/.gitignore

```diff
@@ -0,0 +1,74 @@
+# Miscellaneous
```

Comment:
I think this file can be removed and should be part of the generic .gitignore file

Reply:
You can check that there is a .gitignore in every package folder, as it is used by pub publish to ignore files when publishing too.